core/assumes/sat_checker.go

```diff
@@ -14,7 +14,10 @@ var (
 	// generate better error messages when an "assumes" expression requests a
 	// feature that is not included in the feature set supported by the
 	// current model.
-	userFriendlyFeatureDescriptions = map[string]string{}
+	UserFriendlyFeatureDescriptions = map[string]string{
```

Comment:
Maybe we need an entity here. In the future the amount of information for the assumes could be more complex.

Reply:
This should be good enough for our current use-cases. We can always change this at a later version if we want to display structured information etc.